sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -421,6 +503,39 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
      */
     Azure::Nullable<std::string> Permissions;
 
+    /**
+     * The owner of the file or directory.
+     */
+    Azure::Nullable<std::string> Owner;
+
+    /**
+     * The owning group of the file or directory.
+     */
+    Azure::Nullable<std::string> Group;
+
+    /**
+     * Sets POSIX access control rights on files and directories. Each access control entry (ACE)
+     * consists of a scope, a type, a user or group identifier, and permissions.
+     */
+    Azure::Nullable<std::vector<Models::Acl>> Acls;
+
+    /**
+     * Proposed LeaseId.
+     */
+    Azure::Nullable<std::string> LeaseId;
+
+    /**
+     * Specifies the duration of the lease, in seconds, or InfiniteLeaseDuration for a lease that
+     * never expires. A non-infinite lease can be between 15 and 60 seconds. A lease duration cannot
+     * be changed using renew or change. Does not apply to directories.
+     */
+    Azure::Nullable<std::chrono::seconds> LeaseDuration;
+
+    /**
+     * Optional parameters to schedule the file for deletion.
+     */
+    SchedulePathDeletionOptions ScheduleDeletionOptions;
```

Comment:
Do we need to specify origin type?

Reply:
Yes, because ScheduleDeletion is not supported for a directory, I have tested for this.

Reply:
How do we specify it's relative to creation time or current time, or absolute time?

Reply:
This is aligned with .Net.
If TimeToExpire has value, is relative to current time, because this is create operation, I think current time is also creation time.
If Expires on has value, is absolute time.
The code is following:
```
    AZURE_ASSERT_MSG(
        !(options.ScheduleDeletionOptions.ExpiresOn.HasValue()
          && options.ScheduleDeletionOptions.TimeToExpire.HasValue()),
        "ExpiresOn and TimeToExpire are mutually exclusive.");
    if (options.ScheduleDeletionOptions.ExpiresOn.HasValue())
    {
      protocolLayerOptions.ExpiryOptions
          = Files::DataLake::ScheduleFileExpiryOriginType::Absolute.ToString();
      protocolLayerOptions.ExpiresOn = options.ScheduleDeletionOptions.ExpiresOn.Value().ToString(
          Azure::DateTime::DateFormat::Rfc1123);
    }
    else if (options.ScheduleDeletionOptions.TimeToExpire.HasValue())
    {
      protocolLayerOptions.ExpiryOptions
          = Files::DataLake::ScheduleFileExpiryOriginType::RelativeToNow.ToString();
      protocolLayerOptions.ExpiresOn
          = std::to_string(options.ScheduleDeletionOptions.TimeToExpire.Value().count());
    }
```

Reply:
Maybe I can add some comments about this if this is confusing?

Reply:
I think it's fine.